components/user_settings/display/user_settings_theme/custom_theme_chooser.jsx

```diff
@@ -214,30 +216,34 @@ export default class CustomThemeChooser extends React.PureComponent {
     toggleSidebarStyles = (e) => {
         e.preventDefault();
 
-        $(this.refs.sidebarStylesHeader).toggleClass('open'); // eslint-disable-line jquery/no-class
+        this.refs.sidebarStylesHeader.classList.toggle('open');
         this.toggleSection(this.refs.sidebarStyles);
     }
 
     toggleCenterChannelStyles = (e) => {
         e.preventDefault();
 
-        $(this.refs.centerChannelStylesHeader).toggleClass('open'); // eslint-disable-line jquery/no-class
+        this.refs.centerChannelStylesHeader.classList.toggle('open');
         this.toggleSection(this.refs.centerChannelStyles);
     }
 
     toggleLinkAndButtonStyles = (e) => {
         e.preventDefault();
 
-        $(this.refs.linkAndButtonStylesHeader).toggleClass('open'); // eslint-disable-line jquery/no-class
+        this.refs.linkAndButtonStylesHeader.classList.toggle('open');
         this.toggleSection(this.refs.linkAndButtonStyles);
     }
 
     toggleSection(node) {
-        if (UserAgent.isIos()) {
-            // iOS doesn't support jQuery animations
-            $(node).toggleClass('open'); // eslint-disable-line jquery/no-class
+        node.classList.toggle('open');
+
+        // set overflow after animation, so the colorchooser is fully shown
+        if (node.classList.contains('open')) {
+            setTimeout(() => {
+                node.style.overflowY = 'inherit';
```

Comment:
If you close a section while it's still animating, the contents will overflow/be visible. (Mobile view and tablet/desktop viewports)
![2020-09-28 14 34 47](https://user-images.githubusercontent.com/11724372/94477828-0413df00-0198-11eb-96a7-263055733ca8.gif)

Reply:
Maybe an [ontransitionend](https://developer.mozilla.org/en-US/docs/Web/API/GlobalEventHandlers/ontransitionend) handler would be more appropriate?

Reply:
I changed it to the ontransitionend handler :)